src/content/en/2021/javascript.md

```diff
@@ -674,11 +683,13 @@ The results show that jQuery is the dominant factor in the decrease of vulnerabi
 
 ## How do we use JavaScript?
 
-Now we've looked at how we get the JavaScript, what are we using it for?
+Now that we've looked at how we get the JavaScript, what are we using it for?
 
 ### AJAX
 
-A lot of JavaScript that we write is used to communicate with servers through asynchronous requests to receive information in various formats. AJAX, or asynchronous JavaScript and XML, is typically used to send and receive data in JSON, XML, HTML, text formats.
+A lot of JavaScript that is written is used to communicate with servers through asynchronous requests to receive information in various formats. AJAX, or asynchronous JavaScript and XML, is typically used to send and receive data in JSON, XML, HTML, text formats.
+
+{# TODO - "AJAX, or asynchronous JavaScript and XML," is this a list of 3 or grouped together? #}
```

Comment:
@NishuGoel

Reply:
full form of AJAX so group together, not a list of 3

Reply:
Would such we flip it around and put in italics to show it's introducing a new term:

```
_Asynchronous JavaScript and XML_ (AJAX), is typically used to send and receive data in JSON, XML, HTML, text formats.